spec/support/manager_box.cr

```diff
@@ -0,0 +1,26 @@
+class ManagerBox
+  getter form
+
+  @form = Manager::BaseForm.new
+
+  def initialize
+    name "Mr. Manager"
+  end
+
+  def self.save
+    new.save
+  end
+
+  def save
+    if form.save
```

Comment:
Now you can do `form.save!` and it will raise if it didn't save so you don't need the conditional